dispatches/models/nuclear_case/chgtank.py

```diff
@@ -0,0 +1,590 @@
+**default** - useDefault.
+**Valid values:** {
+**useDefault** - use default package from parent model or flowsheet,
+**PhysicalParameterObject** - a PhysicalParameterBlock object.}"""))
+    config.declare("property_package_args", ConfigBlock(
+        implicit=True,
+        description="Arguments to use for constructing property packages",
+        doc="""A ConfigBlock with arguments to be passed to a property block(s)
+and used when constructing these,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}"""))
+
+
+@declare_process_block_class("CHGTank",
+                             doc="Simple compressed hydrogen gas tank model")
+class CHGTankData(UnitModelBlockData):
+    """
+    Simple compressed hydrogen gas tank model.
+    Unit model to store or supply compressed hydrogen gas.
+    
+    """
+    CONFIG = ConfigBlock()
+    _make_chg_tank_config_block(CONFIG)
+
+    def build(self):
+        """Building model
+        Args:
+            None
+        Returns:
+            None
+        """
+        super(CHGTankData, self).build()
+
+        # Build Control Volume
+        self.control_volume = ControlVolume0DBlock(default={
+            "dynamic": self.config.dynamic,
+            "property_package": self.config.property_package,
+            "property_package_args": self.config.property_package_args})
+
+        # add inlet and outlet states
+        self.control_volume.add_state_blocks(has_phase_equilibrium=False)
+
+        # add tank volume
+        self.control_volume.add_geometry()
+
+        # add pressure balance
+        self.control_volume.add_momentum_balances(
+            balance_type=self.config.momentum_balance_type,
+            has_pressure_change=True)
+
+        # add phase fractions
+        self.control_volume._add_phase_fractions()
+
+        # add a state block 'initial_state' for the storage tank
+        # this state block is needed to compute initial material and energy
+        # holdup using the property package for a given P_init and T_init
+        # NOTE: there is no flow in the initial state so,
+        # flow_mol state variable is fixed to 0
+        self.initial_state = (
+            self.config.property_package.build_state_block(
+                self.flowsheet().config.time,
+                doc="tank initial state",
+                default=self.config.property_package_args))
+        # initial state should not have any flow
+        self.initial_state[:].flow_mol.fix(0)
+
+        # add local lists for easy use
+        phase_list = self.control_volume.properties_in.phase_list
+        pc_set = self.control_volume.properties_in.phase_component_set
+        component_list = self.control_volume.properties_in.component_list
+
+        # Get units from property package
+        units = self.config.property_package.\
+            get_metadata().get_derived_units
+        time_units = units("time")
```

Comment:
These aliases don't really shorten anything - why not just use `units("time")` etc. in the main code.